recipes/ca-certificates/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: ca-certificates
+  version: {{ version }}
+
+# This does not have a source because it pulls these from certifi at present.
+
+build:
+  number: 0
+  skip: true  # [win or not py35]
+  # As openssl includes these (dependency of Python), they will be removed afterwards.
+  # This is our way of ensuring these files are kept anyways. Once conda-forge has its
+  # own openssl, this will no longer be necessary.
+  always_include_files:
+    - ssl/cert.pem
+    - ssl/cacert.pem
+
+requirements:
+  build:
+    - python
+    - certifi {{ version }}
```

Comment:
Our versioning matches `certifi`'s exactly as we are just getting our certs from there and relocating them. Currently, this is the simplest and most effective way to get this done.

In the future, we might generate these ourselves. We could do this by using `nss`, which we have a WIP recipe for ( https://github.com/conda-forge/staged-recipes/pull/536 ). Though I have no plans of pushing that approach at present.